packages/storybook/src/nimble/rich-text-mention/users/rich-text-mention-users.stories.ts

```diff
@@ -55,24 +60,38 @@ export const richTextMentionUsers: StoryObj = {
     argTypes: {
         pattern: {
             description: patternDescription,
-            control: false
+            control: false,
+            table: { category: apiCategory.attributes },
         },
         checkValidity: {
             name: 'checkValidity()',
-            description:
-                'Returns `true` if the mention configuration is valid, otherwise `false`.'
+            description: checkValidityDescription({ componentName: 'rich text mention users' }),
+            table: { category: apiCategory.methods },
         },
         validity: {
-            description: mappingUserValidityDescription
+            description: mappingUserValidityDescription,
+            table: { category: apiCategory.nonAttributeProperties },
         },
         mentionUpdate: {
             name: 'mention-update',
-            description: mentionUpdateEventDescription
+            description: mentionUpdateEventDescription,
+            table: { category: apiCategory.events },
         },
         buttonLabel: {
             name: 'button-label',
             description:
-                'Label and title text for the mention button in the footer toolbar.'
+                'Label and title text for the mention button in the footer toolbar.',
+            table: { category: apiCategory.attributes },
+        },
+        content: {
+            name: 'default',
+            description: `Configure how users are displayed by adding \`${mappingTextTag}\` elements as content. Ensure that each user mentioned in the markdown input has a corresponding user mapping.
+
+For the rich text editor, the user mappings are used to populate the mention dropdown. Update the mention elements dynamically by listening to the \`mention-update\` event and filtering the user mappings based on the current text input.
+It is recommended to limit the number of users displayed to 50 or fewer.
+
+For more details, see [Client Usage Guidance on Filtered Users](https://github.com/ni/nimble/blob/main/packages/nimble-components/src/rich-text/specs/mention-hld.md#client-usage-guidance-on-filtered-users).`,
```

Comment:
I think the ultimate source of truth should be these docs, can we inline it here instead?